### PR TITLE
Documentation for ablator/main/proto.py

### PR DESCRIPTION
**Pull request by hieuchi911**
Documentation for ablator/main/proto.py

### CODE REVIEW

File: ablator/main/proto.py [hunk 1]
```diff
@@ -32,14 +54,29 @@ def pre_train_setup(self):
...
 
     def launch(self, debug: bool = False):
+        """
+        Initialize the data state of the wrapper and train the model inside the wrapper, then sync training
+        results (logged to experiment directory while training) with external logging services (e.g Google
+        cloud storage, other remote servers).
+
+        Parameter
+        ---------
+        debug : bool, default=True
```
Comment (hjzccc) [inline ablator/main/proto.py]: default is false

File: ablator/main/proto.py [hunk 2]
```diff
@@ -32,14 +54,29 @@ def pre_train_setup(self):
...
+        Initialize the data state of the wrapper to force downloading and processing any data artifacts
         in the main train process as opposed to inside the wrapper.
         """
         self.pre_train_setup()
         mock_wrapper = copy.deepcopy(self.wrapper)
         mock_wrapper._init_state(run_config=copy.deepcopy(self.run_config), debug=True)
 
     def launch(self, debug: bool = False):
+        """
+        Initialize the data state of the wrapper and train the model inside the wrapper, then sync training
```
Comment (hjzccc) [inline ablator/main/proto.py:66]: should we add comments for sync here? since sync does nothing in proto trainer

Comment (MrLYG) [reply ablator/main/proto.py:66]: I think it will be added after

**Review by MrLYG — COMMENTED** [review]
looks great

File: ablator/main/proto.py (hunk 2 above)
Comment (hieuchi911) [reply ablator/main/proto.py:66]: It's just a prototype class so I think it's not important whether to mention sync or not.

File: ablator/main/proto.py (hunk 1 above)
Comment (hieuchi911) [reply ablator/main/proto.py]: Fixed

**Review by MrLYG — APPROVED** [review]

**Review by hjzccc — APPROVED** [review]

**Review by fostiropoulos — APPROVED** [review]

**Pull request merged**